locales/ar.json

```diff
@@ -73,6 +73,7 @@
     "preferences_thin_mode_label": "الوضع الخفيف: ",
     "preferences_category_misc": "تفضيلات متنوعة",
     "preferences_automatic_instance_redirect_label": "إعادة توجيه المثيل التلقائي (إعادة التوجيه إلى redirect.invidious.io): ",
+    "preferences_include_youtube_links_label": "عرض روابط إعادة التوجيه على يوتيوب: ",
```

Comment:
You only need to provide translations for en-US. 

Other translations are done through weblate (ex: https://github.com/iv-org/invidious/pull/4989/files ).